src/Microsoft.AspNet.Mvc.Xml/RequiredValidationHelper.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+using System.Runtime.Serialization;
+using Microsoft.AspNet.Mvc.ModelBinding;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+
+
+
+
+    /// <summary>
+    /// Validates types having value type properties decorated with <see cref="RequiredAttribute"/>
+    /// but no <see cref="DataMemberAttribute"/>.
+    /// </summary>
```

Comment:
Would be good to add a comment here or remarks saying **why** this is important and what the impact is.